src/python/grpcio_test/grpc_test/framework/face/testing/future_invocation_asynchronous_event_service_test_case.py

```diff
@@ -306,9 +308,10 @@ def testParallelInvocations(self):
         second_request = test_messages.request()
 
         first_response_future = self.stub.future_value_in_value_out(
-            name, first_request, _TIMEOUT)
+            name, first_request, test_constants.SHORT_TIMEOUT)
+        # TODO(bug 2039): use SHORT_TIMEOUT instead
```

Comment:
I don't understand this - would working around #2039 mean wanting to use LONG_TIMEOUT but having to use SHORT_TIMEOUT instead?

Is this something that failed when you ran it in draft with LONG_TIMEOUT in both places in this method body?